interview-questions.md

```diff
@@ -8,36 +8,61 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-Your answer:
+Your answer: A solid method of solving this issue is editing the schema of students and then migrating said database. The content to edit within the schema should be an added key that you could call "cohort" or something along the same lines thats value is an integer. Only after running the command db:merge will the new key in the database of students be made available and then, if database items are already created in students, values will need to be reassigned using active record queries or some kind of database manager like postman to update the data. If this is not done, the foreign key will be initialized with the nil value because they are initially empty.
 
-Researched answer:
+Researched answer: After doing some further research there is a quick migration command you can run in terminal that will allow you to run a migration adding keywords to get what you need. You can also use the command rake db:rollback. From here the latest migration (in which the foreign key wasn't added) will be reverted which will allow you to edit your change table with the add_column keyword. From there you can pass it <somename>_id which will be interpreted as the foreign key.
```

Comment:
nice research. On our laptops, we will use rails db:rollback. 
What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?